tests/cli/test_rasa_interactive.py

```diff
@@ -61,7 +62,7 @@ def test_pass_arguments_to_rasa_train(
 
     # Mock actual training
     mock = Mock()
-    monkeypatch.setattr(rasa, "train", mock.method)
+    monkeypatch.setattr(train, "rasa_train", mock.method)
```

Comment:
This change is wrong. It mocks the actual `train` function from the `rasa.train` module. 
It was not the best idea back then to have functions with the same name as modules on the same level 😬

Reply:
I'm confused, what should I do here?

Reply:
Revert the change to `"train"` . Seems you already did that 💯